### PR TITLE
Added 'rhinoder' and 'valideno' to database.json

### PR DESCRIPTION
**Pull request by Caesar2011**

### CODE REVIEW

File: database.json [hunk 1]
```diff
@@ -2214,6 +2214,12 @@
     "repo": "resulty",
     "desc": "Provides simple, Rust-like Result and Option objects for Deno."
   },
+  "rhinoder": {
+    "type": "github",
+    "owner": "Caesar2011",
+    "repo": "rhinoder",
+    "desc": "Reloads a deno application on every file change"
+  },
```
Comment (alirealasad) [inline database.json:2222]: put the repo in an alphabetic manner

i.e. after the ripemd160 module & before the router module.

that will solve the CI error.

Comment (Caesar2011) [reply database.json:2222]: But `h` comes before `i` in the alphabet? I am so confused right now!

Comment (alirealasad) [reply database.json]: ooh sorry just put in right place https://github.com/denoland/deno_website2/pull/882/checks?check_run_id=717111435

**Comment by Caesar2011** [comment]
Will open a new and clean branch

**Pull request closed without merging**